src/com/osreboot/ridhvl2/HvlMath.java

```diff
@@ -18,6 +20,30 @@ private HvlMath(){}
 	 */
 	public static final float PI = (float)Math.PI;
 
+	/**
+	 * A {@linkplain Random} object used for random number generation.
+	 */
+	private static Random seed = new Random();
+
+	/**
+	 * Returns a random <code>int</code> falling between <code>minArg</code> and
+	 * <code>maxArg</code>, inclusive.
+	 * 
+	 * @param minArg the smallest possible <code>int</code> value that can be returned
+	 * @param maxArg the largest possible <code>int</code> value that can be returned
+	 * @return A random <code>int</code> between these two points
+	 */
+	public static int randomIntBetween(int minArg, int maxArg) {
+		maxArg++;
+		if(maxArg > minArg) {
+			return minArg + seed.nextInt(maxArg - minArg);
+		}
+		if(maxArg < minArg) {
+			return maxArg + seed.nextInt(minArg - maxArg);
+		}
+		return minArg;
+	}

```

Comment:
Please verify inclusivity/exclusivity by running this method a bunch of times in a test project.